ExtCore/Control.Indexed.Result.fs

```diff
@@ -0,0 +1,341 @@
+(*
+
+Copyright 2010-2012 TidePowerd Ltd.
+Copyright 2013 Jack Pappas
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*)
+
+//
+namespace ExtCore.Control.Indexed
+
+open ExtCore
+
+
+(*** Workflow Monoids ***)
+
+
+/// <summary>
+/// </summary>
+/// <typeparam name="S1"></typeparam>
+/// <typeparam name="S2"></typeparam>
+/// <typeparam name="T"></typeparam>
+/// <typeparam name="Error"></typeparam>
```

Comment:
Remove all such meaningless XML Docs?

Reply:
Could they have been added in order to silence warnings about missing xml docs?

Reply:
maybe. if so, it smells like try..with _ -> ()

Reply:
some of these things could be improved by a little bit of documentation, for instance It's a bit confusing to read the CPS function definitions

Reply:
I added these empty XML docs to remind myself -- years ago -- to fill them in so new users would get some helpful hints from Intellisense. Obviously, I still haven't gotten around to doing it, but I think Oskar's right -- it'd be helpful to users to have the information rather than just removing the comments.